spec/policies/dashboard_policy_spec.rb

```diff
@@ -18,8 +18,8 @@
 
   context 'permission for change_status' do
     it 'grant access to admin user' do
-      policy = described_class.new(admin_user,nil)
-      expect(policy.change_status?).to be true
+      policy = described_class.new(admin_user,admin_user)
```

Comment:
two admin user? can you please describe why?

Reply:
In the change_status? policy, if a user is an admin, he can't change the status of him or any other admin. In our policy class, the first argument is the current user, and the second argument is which record you want to authorize. 

In this case, I check if a user is an admin; he can't change his or any other admin member status.
For this reason, first argument is admin_user and also pass second argument is admin_user

Reply:
okay! please add a space after comma.